scripts/repo-health-script.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+set -e -v
+
+# Click requires this to work cause it interfaces weirdly with python 3 ASCII default
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+
+WORKSPACE=$PWD
+
+# If the REPORT_DATE variable is set and not an empty string parse the date to standardize it.
+if [[ ! -z $REPORT_DATE ]]; then REPORT_DATE=$(date '+%Y-%m-%d' -d "$REPORT_DATE"); fi
+
+##########################################
+# Get list of repos in given organizations
+##########################################
+
+pip-sync -q repo_tools/requirements/base.txt
+pip install -q -e repo_tools
+cd $WORKSPACE
+touch "repositories.txt"
+for ORG_NAME in ${ORG_NAMES[@]}; do
+  get_org_repo_urls "${ORG_NAME}" --url_type https --forks --add_archived --output_file "repositories.txt" --username "${GITHUB_USER_EMAIL}" --token "${GITHUB_TOKEN}" --ignore-repo "${REPOS_TO_IGNORE}"
```

Comment:
Long lines should be split across lines with backslashes so they can be read more easily.